kaggle-classification/hparam_config.yaml

```diff
@@ -22,10 +23,15 @@ trainingInput:
       - parameterName: num_filters
         type: INTEGER
         minValue: 10
-        maxValue: 30
+        maxValue: 200
         scaleType: UNIT_LINEAR_SCALE
       - parameterName: dropout_keep_prob
         type: DOUBLE
         minValue: 0.5
         maxValue: 1
-        scaleType: UNIT_LINEAR_SCALE
\ No newline at end of file
+        scaleType: UNIT_LINEAR_SCALE
+      - parameterName: learning_rate
+        type: DOUBLE
+        minValue: 0.000001
+        maxValue: 0.1
+        scaleType: UNIT_LOG_SCALE
```

Comment:
Can you add a newline to the end of the file?

Reply:
Added.